src/model/highlight-list.v3.yaml

```diff
@@ -0,0 +1,91 @@
+                                            type: string
+                                            enum:
+                                              - blog-article
+                                  - $ref: ../snippets/blog-article.v1.yaml
+                              - allOf:
+                                  - properties:
+                                        type:
+                                            type: string
+                                            enum:
+                                              - collection
+                                  - $ref: ../snippets/collection.v1.yaml
+                              - allOf:
+                                  - properties:
+                                        type:
+                                            type: string
+                                            enum:
+                                              - digest
+                                  - $ref: ../snippets/digest.v1.yaml
+                              - allOf:
+                                  - properties:
+                                        type:
+                                            type: string
+                                            enum:
+                                              - event
+                                  - $ref: ../snippets/event.v1.yaml
+                              - allOf:
+                                  - properties:
+                                        type:
+                                            type: string
+                                            enum:
+                                              - interview
+                                  - $ref: ../snippets/interview.v1.yaml
+                              - allOf:
+                                  - properties:
+                                        type:
+                                            type: string
+                                            enum:
+                                              - labs-post
+                                  - $ref: ../snippets/labs-post.v1.yaml
+                              - allOf:
+                                  - properties:
+                                        type:
+                                            type: string
+                                            enum:
+                                              - podcast-episode
+                                  - $ref: ../snippets/podcast-episode.v1.yaml
+                              - $ref: ../misc/podcast-episode-chapter.v1.yaml
+                              - allOf:
+                                  - properties:
+                                        type:
+                                            type: string
+                                            enum:
+                                              - press-package
+                                  - $ref: ../snippets/press-package.v1.yaml
```

Comment:
The only difference between `highlight-list.v2.yaml` and `highlight-list.v3.yaml` is the addition of the support for `press-package`.

See: https://github.com/elifesciences/api-raml/pull/236/files#diff-5b5d735f3567c09e9e0068fb93588f71R79-R85